SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/TestActors/GASTestActorBase.h

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "Abilities/GameplayAbility.h"
+#include "AbilitySystemComponent.h"
+#include "AbilitySystemInterface.h"
+#include "CoreMinimal.h"
+#include "SpatialGDKFunctionalTests/SpatialGDK/TestActors/ReplicatedTestActorBase.h"
```

Comment:
```suggestion
#include "SpatialGDK/TestActors/ReplicatedTestActorBase.h"
```

Reply:
The shortened path isn't actually valid, though I'm not entirely sure why that is. However since `ReplicatedTestActorBase.h` is adjacent to this file I can actually just include it without any prefix.